alpine-common/src/main/java/alpine/Config.java

```diff
@@ -101,6 +101,7 @@ public enum AlpineKey implements Key {
         WORKER_THREAD_MULTIPLIER               ("alpine.worker.thread.multiplier",   4),
         DATA_DIRECTORY                         ("alpine.data.directory",             "~/.alpine"),
         SECRET_KEY_PATH                        ("alpine.secret.key.path",            null),
+        SECRET_API_KEYPREFIX                   ("alpine.secret.api.keyprefix",       "dta_"),
```

Comment:
I'd suggest to rename this to something like `alpine.api.key.prefix` to avoid confusion with the secret key, which is used for encryption.